README.md

```diff
@@ -172,4 +172,28 @@ If you want to end the connection early, you can!
 streaming_client.end()
 ```
 
-Otherwise, the connection will end when the server obtains an "EOS" message.
\ No newline at end of file
+Otherwise, the connection will end when the server obtains an "EOS" message.
+
+### Submitting Custom Vocabularies
+
+In addition to passing custom vocabularies as parameters in the async api client, you can create and view your custom vocabularies independently.
```

Comment:
This statement is misleading.

1. You cannot view the custom vocabulary in the way that the customer is probably expecting. You can only view the CV processing status and the metadata, not the CV phrases themselves.
2. There is little motivation for a customer to presubmit CV for async jobs. The primary motivation for presubmitting CV is to use CV with streaming. I think your description should make this clear.